### DISCUSSION TITLE
DS920+ MaxDisks issue

### DISCUSSION THREAD

**Discussion by GunterO** [pocopico/tinycore-redpill#301] [Q&A]
Is there a way to extend the nr of disks?
Thinks I tried:
Altered /etc/synoinfo.conf & /etc.defaults/synoinfo.conf
maxdisks="16"
internalportcfg="0xffff"
Checked user_config.json, and looks good:
  "synoinfo": {
    "internalportcfg": "0xffff",
    "maxdisks": "16",
    "support_bde_internal_10g": "no",
    "support_disk_compatibility": "no",
    "support_memory_compatibility": "no"
  },

DSM is showing this:
![image](https://github.com/pocopico/tinycore-redpill/assets/7931134/d364195c-f996-4efa-87c1-814f4ebdedad)

However, the 5 drives are visible:
![image](https://github.com/pocopico/tinycore-redpill/assets/7931134/18c6e895-9287-4da0-89c3-ae6280ce0193)

So, is this just a GUI issue?

**Comment by pocopico** [comment]
what model are  you using ? there is always a way

**Reply by GunterO** [reply]
DS920+ on a baremetal Asus Q170M-C with Intel i7-6700 cpu.

**Reply by yyxida** [reply]
DS920 uses DTS (Device Tree Source),satamap is invalid for it

**Comment by pocopico** [comment]
How many disks do you have in total ?

**Reply by GunterO** [reply]
Currently 4 HDD's (for SHR) an one SSD. But I want to add more HDD's to the SHR later, and I'm worried that it won't work because it is only showing 4 drives (see screenshots).
